client/post-editor/editor-sharing/publicize-message.scss

```diff
@@ -1,27 +1,10 @@
 .editor-sharing__publicize-message {
-	.editor-sharing__message-heading {
-		@extend .editor-drawer__heading;
-
-		font-size: 14px;
-	}
-
-	.editor-sharing__message-description {
-		@extend .editor-drawer__description;
-		.gridicon {
-			margin: 0 4px -1px 0;
-		}
-	}
-
-	.editor-sharing__message-input {
-		margin: 8px 0;
-	}
```

Comment:
Note: this was required because `PublicizeMessage` in the drawer didn't use the `EditorDrawerLabel` which has vertical margins already configured.